src/dmd/cparse.d

```diff
@@ -1666,6 +1666,159 @@ final class CParser(AST) : Parser!AST
         }
     }
 
+    /*************************************
+     * C11 6.7.2
+     * member-declaration:
```

Comment:
This grammar is not in C11. It needs to match the C11 spec. There is no `member-declaration` in the spec.

Reply:
http://www.open-std.org/jtc1/sc22/wg14/www/docs/n2454.pdf